v0.3/cruorg/components/accordion/accordion.scss

```diff
@@ -2,3 +2,158 @@
 @import '../../branding-imports';
 
 /* Accordion styles */
+
+@mixin cru-accordion {
+    &__button { 
+        @include cru-button;
```

Comment:
@sebamelzi This is not the right place to use this mixin. Although this class is used on a button tag, its design doesn't match our standard buttons.